server/svix-server/src/v1/endpoints/endpoint/crud.rs

```diff
@@ -21,29 +21,31 @@ use crate::{
     db::models::{application, endpoint, endpointmetadata, eventtype},
     error::{HttpError, Result, ValidationErrorItem},
     v1::utils::{
+        apply_pagination_asc,
         patch::{patch_field_non_nullable, UnrequiredField, UnrequiredNullableField},
-        ApplicationEndpointPath, EmptyResponse, ListResponse, ModelIn, ModelOut, Pagination,
-        PaginationLimit, ValidatedJson, ValidatedQuery,
+        ApplicationEndpointPath, EmptyResponse, ListResponse, ModelIn, ModelOut, NewestFirst,
+        Pagination, PaginationLimit, ReversibleIterator, ValidatedJson, ValidatedQuery,
     },
     AppState,
 };
 use hack::EventTypeNameResult;
 
 pub(super) async fn list_endpoints(
     State(AppState { ref db, .. }): State<AppState>,
-    pagination: ValidatedQuery<Pagination<EndpointId>>,
+    ValidatedQuery(pagination): ValidatedQuery<Pagination<ReversibleIterator<EndpointId>>>,
+    ValidatedQuery(NewestFirst { newest_first }): ValidatedQuery<NewestFirst>,
```

Comment:
Also, I think we can probably come up with a better name. It was just a terrible place-holder I suggested. :P

Reply:
Renamed it to `order=ascending`/`order=descending`.